lib/themes/dosomething/paraneue_dosomething/js/components/SlideshowController.js

```diff
@@ -1,14 +1,26 @@
+import { sendEvent } from '../utilities/Analytics';
+
 const React = require('react');
 
 /**
  * SlideshowController Component
  * <SlideshowController />
  */
 class SlideshowController extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.finish = this.finish.bind(this);
+  }
+
+  finish() {
```

Comment:
jumping in here cuz ive been eavesdropping! i agree that using more standard components moving forwarad, but if it's at all possible to sit down with whoever might be the creator of the existing events/know the most about them and document what they're tracking. I know it was a struggle when digging into event data for lose your v card